lib/redfish_api.py

```diff
@@ -263,28 +274,15 @@ def get_openapi_from_redfish_org(self):
         Method to get openapi.yaml schema file from openapi url of redfish organisation
         :return: Bool,string output for openapi schema file
         """
-
         try:
-            #if len(self.openapi_dict) > 0:
-            #    pass
-            #    #return
-            filename = self.config_dict["filename"]
-            if os.path.exists(filename) != True:
-                print("Not exists", self.config_dict["openapi_url"])
-                result = requests.get(self.config_dict["openapi_url"])
-                if result.status_code == 200:
```

Comment:
Are we changing the logic here? 
Previously we were checking if open api yaml file exists or not , if not then download the file.

Reply:
Added back the logic in latest commit